ClosedXML/Excel/Drawings/IXLMarker.cs

```diff
@@ -1,10 +0,0 @@
-using System.Drawing;
-
-namespace ClosedXML.Excel.Drawings
-{
-    internal interface IXLMarker
```

Comment:
Out of curiosity, why did you decide to remove the interface?

Reply:
Neither `IXLMarker` nor `XLMarker` is public. Both are internal, so I don't really see the point of an interface. Or am I missing something?

Reply:
I don't see any problems with this either. We can add the interface back any time if we need it.